products/angularjs.md

```diff
@@ -91,3 +102,5 @@ on the final version 1.8.3 which was released April 7, 2022.
 Beginning January 1, 2022, [extended long term support for AngularJS](https://xlts.dev/angularjs)
 became available for 1.5.x and 1.8.x versions through a fork of the project maintained by the
 independent team at [XLTS.dev](http://xlts.dev/).
+
+AngularJS version 1.8 has continued commercial support through the XLTS 1.9 forked version where XLTS 1.9 is a compatible replacement for AngularJS version 1.8.
```

Comment:
This paragraph could be merged with the previous one, will do it in a follow-up update.